web/src/main/resources/swagger/swagger_en.properties

```diff
@@ -20,6 +20,7 @@ swagger.mscore.institution.relationships=returns the relationships related to th
 swagger.mscore.onboarding.operator=performs operators onboarding on an already existing institution
 swagger.mscore.onboarding.subdelegates=performs subdelegates onboarding on an already existing institution
 swagger.mscore.onboarding.users=The service adds users to the registry if they are not present and associates them with the institution and product contained in the body
+swagger.mscore.onboarding.users.delete=The service delete the users changing the user status to 'DELETE' and remove from DB the PecNotification record by productId and institutionId
```

Comment:
you could edit "The service delete the users changing the user status to 'DELETE' with reference about update on onboarding record.....for ex: "The service set DELETE status on onboarding association with the product and ..."